### PR TITLE
Fixes #1512: ImageClickInput split-screen test cases 

### LINKED ISSUE #1512: ImageClickInput test cases
Add test cases for `ImageClickInput` interaction for the split-screen case in Exploration/Question Player

### PR DESCRIPTION
**Pull request by MohamedMedhat1998**
## Explanation

- Fixes #1512 
- Added test cases for ImageClickInput for split-screen
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/test/java/org/oppia/app/testing/player/split/PlayerSplitScreenTesting.kt [hunk 1]
```diff
@@ -79,4 +78,34 @@ class PlayerSplitScreenTesting {
       }
     }
   }
+
+  @Test
+  @Config(qualifiers = "w540dp-h960dp-xhdpi") // 5.5 (inch)
+  fun testSplitScreen_540x960_xhdpi_imageClickInput_NoSplit() {
```
Comment (rt4914) [inline app/src/test/java/org/oppia/app/testing/player/split/PlayerSplitScreenTesting.kt]: Fix all test case names in this screen. Suggested change:
`_NoSplit` -> `_noSplit`
`_Split` -> `_split`

**Review by rt4914 — CHANGES_REQUESTED** [review]
LGTM, one nit change.

File: app/src/test/java/org/oppia/app/testing/player/split/PlayerSplitScreenTesting.kt (hunk 1 above)
Comment (MohamedMedhat1998) [reply app/src/test/java/org/oppia/app/testing/player/split/PlayerSplitScreenTesting.kt]: Done

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks.

**Pull request merged**